docs/dictionary/property/executionContexts.lcdoc

```diff
@@ -38,7 +38,8 @@ A common use of the <executionContexts> is to obtain the name of the
 object and handler that called the current handler, this information is
 available as: line -2 of the executionContexts.
 
+Note that if the function call occurred in a behavior script, then the line number refers to the behavior script, not the target script.
```

Comment:
@mwieder we usually use `>**Note:**` to present things like this in a block quote.